kubernetes/cluster_deploy.go

```diff
@@ -45,26 +45,26 @@ func (c *Cluster) Deploy(options *ltops.DeployOptions) error {
 	}
 
 	if previouslyDeployed {
-		cmd := exec.Command("helm", "upgrade", "-f", configFileLocation, c.Release(), "mattermost/mattermost-helm")
+		cmd := exec.Command("helm", "upgrade", "-f", configFileLocation, c.Release(), "mattermost/mattermost-enterprise-edition")
 		if out, err := cmd.CombinedOutput(); err != nil {
 			return errors.Wrap(err, "unable to install mattermost chart, error from helm: "+string(out))
 		}
 
 		// Delete the app pods to make sure they reset caches and restart to pick up config changes
-		cmd = exec.Command("kubectl", "delete", "po", "-l", fmt.Sprintf("release=%v,app=mattermost-helm", c.Release()))
+		cmd = exec.Command("kubectl", "delete", "po", "-l", fmt.Sprintf("release=%v,app=mattermost-enterprise-edition", c.Release()))
 		if out, err := cmd.CombinedOutput(); err != nil {
 			return errors.Wrap(err, "unable to restart app server pods, error from kubectl: "+string(out))
 		}
 
 		// Delete the loadtest pods to make sure they pick up config changes
-		cmd = exec.Command("kubectl", "delete", "po", "-l", fmt.Sprintf("release=%v,app=mattermost-helm-loadtest", c.Release()))
+		cmd = exec.Command("kubectl", "delete", "po", "-l", fmt.Sprintf("release=%v,appmattermost-enterprise-edition-loadtest", c.Release()))
```

Comment:
Is this right? `appmattermost-enterprise-edition-loadtest` vs `app=mattermost-enterprise-edition-loadtest`

Reply:
i will check and rethink :)

Reply:
fixed @lieut-data